base/shell/explorer/taskswnd.cpp

```diff
@@ -491,7 +491,7 @@ class CTaskSwitchWnd :
     {
         HICON hIcon = NULL;
 #define GET_ICON(type) \
-    SendMessageTimeout(hwnd, WM_GETICON, (type), 0, SMTO_ABORTIFHUNG, 100, (PDWORD_PTR)&hIcon)
+    SendMessageTimeout(hwnd, WM_GETICON, (type), 0, SMTO_NOTIMEOUTIFNOTHUNG, 100, (PDWORD_PTR)&hIcon)
```

Comment:
Is this change required?

Reply:
Yes, this the most significant change that fixes the issue.